go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/maticnetwork/heimdall
 
-go 1.12
+go 1.16
```

Comment:
For any reviewer, we have bumped the Go version to use the new Go embed feature.

Reply:
Build is failing because build workflow uses 1.13 version eq:
```with:
          go-version: 1.13
```